app/home/templates/home.tpl.jade

```diff
@@ -1,9 +1,21 @@
-div(layout='column', layout-fill='')
-      md-toolbar#navigation
-        .md-toolbar-tools
-          span(flex="80") 
-            img(src="images/cognizant-logo.png")
-          md-button(flex="6", ui-sref-active="active", data-ui-sref='home.login') LogIn
-          md-button(flex="6", ui-sref-active="active", data-ui-sref='home.register') Register
-      md-content
-        ui-view
\ No newline at end of file
+md-toolbar#navigation
+  .md-toolbar-tools
```

Comment:
always keep in mind the less things in scope the better.
And always favor static over dynamic behaviour